integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/CarbonGlobalDictionaryRDD.scala

```diff
@@ -297,7 +297,8 @@ class CarbonBlockDistinctValuesCombineRDD(
           val complexDelimiters = new util.ArrayList[String]
           model.delimiters.foreach(x => complexDelimiters.add(x))
           for (i <- 0 until dimNum) {
-            dimensionParsers(i).parseString(CarbonScalaUtil.getString(row.get(i),
+            dimensionParsers(i).parseString(CarbonScalaUtil.getString(row,
```

Comment:
move ` CarbonScalaUtil.getString(row,` to next line like `CarbonScalaUtil.getString(row, i),`

Reply:
done